doc/modules/computing.rst

```diff
@@ -376,7 +516,71 @@ In the case of sparse input (particularly in ``CSR`` format), it is generally
 sufficient to not generate the relevant features, leaving their columns empty.
 
 Links
------
+......
 
   - `scikit-learn developer performance documentation <../developers/performance.html>`_
   - `Scipy sparse matrix formats documentation <http://docs.scipy.org/doc/scipy/reference/sparse.html>`_
+
+Parallelism, resource management, and configuration
+=====================================================
+
+.. _parallelism:
+
+Parallel and distributed computing
+-----------------------------------
+
+Scikit-learn uses the `joblib <https://joblib.readthedocs.io/en/latest/>`__
+library to enable parallel computing inside its estimators. See the
+joblib documentation for the switches to control parallel computing.
+
+Note that, by default, scikit-learn uses its embedded (vendored) version
+of joblib. A configuration switch (documented below) controls this
+behavior.
+
+Configuration switches
+-----------------------
+
+Python runtime
+..............
+
+:func:`sklearn.set_config` controls the following behaviors:
+
+:assume_finite:
+
+    used to skip validation, which enables faster computations but may
+    lead to segmentation faults if the data contains NaNs.
+
+:working_memory:
+
+    the optimal size of temporary arrays used by some algoritms.
+
+Environment variables
+......................
+
+These environment variables should be set before importing scikit-learn.
+
+:SKLEARN_SITE_JOBLIB:
+
+    When this environment variable is set to a non zero value,
+    scikit-learn uses the site joblib rather than its vendored version.
+    Consequently, joblib must be installed for scikit-learn to run
```

Comment:
You don't think it's worth mentioning that Memory and dumped content may be invalidated when switching?